migration/migrating_4_1_4/deploying_cam.adoc

```diff
@@ -1,25 +1,66 @@
 include::modules/common-attributes.adoc[]
 :context: migrating-4_1-4_x
 :migrating-4_1-4_x:
 
 toc::[]
 
-Deploying the Cluster Application Migration (CAM) tool requires installing the Cluster Application Migration Operator on the xref:../../migration/migrating_4_1_4/deploying_cam.adoc#installing-cam-operator-ocp-4_sourcecluster-4_1-4_x[{product-title} 4.1 source] and xref:../../migration/migrating_4_1_4/deploying_cam.adoc#installing-cam-operator-ocp-4_targetcluster-4_1-4_x[{product-title} {product-version} target] clusters.
+You can install the Cluster Application Migration Operator on your {product-title} {product-version} target cluster and 4.1 source cluster. The Cluster Application Migration Operator installs the Cluster Application Migration (CAM) tool (CAM web console and Migration controller) on the target cluster by default.
 
-The Cluster Application Migration Operator installs the CAM tool (CAM web console and Migration controller) on the target cluster by default. You can link:https://access.redhat.com/articles/5064151[install the CAM tool on {product-title} 3 and remote clusters].
+[NOTE]
+====
+Optional: You can configure the Cluster Application Migration Operator to install the CAM tool link:https://access.redhat.com/articles/5064151[on an {product-title} 3 cluster or on a remote cluster].
+====
 
-:context: sourcecluster-4_1-4_x
-:sourcecluster-4_1-4_x:
-include::modules/migration-installing-cam-operator-ocp-4.adoc[leveloffset=+1]
-:sourcecluster-4_1-4_x!:
+In a restricted environment, you can install the Cluster Application Migration Operator from a local mirror registry.
+
+After you have installed the Cluster Application Migration Operator on your clusters, you can launch the CAM tool.
+
+[id='installing-cam-operator_{context}']
+== Installing the Cluster Application Migration Operator
+
+You can install the Cluster Application Migration Operator with the Operation Lifecycle Manager (OLM) on an {product-title} {product-version} target cluster and on an {product-title} 4.1 source cluster.
+
+:!migrating-4_1-4_x:
 
 :context: targetcluster-4_1-4_x
 :targetcluster-4_1-4_x:
-include::modules/migration-installing-cam-operator-ocp-4.adoc[leveloffset=+1]
+include::modules/migration-installing-cam-operator-ocp-4.adoc[leveloffset=+2]
 :targetcluster-4_1-4_x!:
 
-include::modules/migration-launching-cam.adoc[leveloffset=+1]
+:context: sourcecluster-4_1-4_x
+:sourcecluster-4_1-4_x:
+include::modules/migration-installing-cam-operator-ocp-4.adoc[leveloffset=+2]
+:sourcecluster-4_1-4_x!:
 
+:context: disconnected-4-1-4
+:disconnected-4-1-4:
+[id='installing-cam-operator_{context}']
+== Installing the Cluster Application Migration Operator in a restricted environment
+
+You can build a custom Operator catalog image for {product-title} 4, push it to a local mirror image registry, and configure the Operation Lifecycle Manager to install the Cluster Application Migration Operator from the local registry.
```

Comment:
Is "4" a specific enough version number?

Reply:
yes